packages/db/src/schema/wallet-schema.ts

```diff
@@ -5,6 +5,7 @@ import { walletTypeSchema } from './wallet-type-schema'
 export const walletSchema = z.object({
   accountId: z.string(),
   createdAt: z.date(),
+  derivationIndex: z.number().default(0),
```

Comment:
The `derivationIndex` must be stored in the `Wallet` so we can figure out know what the 'next' wallet derivation index is.